xbmc/guilib/GUIControl.cpp

```diff
@@ -403,6 +403,8 @@ bool CGUIControl::CanFocus() const
 bool CGUIControl::IsVisible() const
 {
   if (m_forceHidden) return false;
+  if (m_transform.alpha <= 0.01f)
```

Comment:
Maybe combine with the condition above instead of adding another if/branch?

Reply:
If you want. I've did it in a similar style as above.

Reply:
Done.